blueprints/instance-initializer-test/files/tests/unit/instance-initializers/__name__-test.js

```diff
@@ -0,0 +1,23 @@
+import Ember from 'ember';
+import { initialize } from '<%= dependencyDepth %>/instance-initializers/<%= dasherizedModuleName %>';
+import { module, test } from 'qunit';
+
+var appInstance, application;
+
+module('<%= friendlyTestName %>', {
+  beforeEach: function() {
+    Ember.run(function() {
+      application = Ember.Application.create();
+      appInstance = application.__deprecatedInstance__;
```

Comment:
@cibernox Re: who has this knowledge in his head, do the following links help at all?

https://github.com/emberjs/website/pull/2083
https://github.com/emberjs/website/pull/1951

Looks like @dgeb and @tomdale did most of the work on this ...